ostree/ostree_dest.go

```diff
@@ -0,0 +1,255 @@
+package ostree
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"strconv"
+	"fmt"
+	"path/filepath"
+
+	"encoding/json"
+	"github.com/containers/image/manifest"
+	"github.com/containers/storage/pkg/archive"
+	"github.com/containers/image/types"
+	"github.com/opencontainers/go-digest"
+	"github.com/pkg/errors"
+)
+
+type blobToImport struct {
+	Size      int64
+	Digest    digest.Digest
+	BlobPath  string
+}
+
+type descriptor struct {
```

Comment:
We're duplicating this too many. Worth using the official image-spec descriptor type